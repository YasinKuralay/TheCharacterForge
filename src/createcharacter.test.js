import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import CreateCharacter from "./createcharacter.js";

it("renders the CreateCharacter component without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<CreateCharacter />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it("matches the previous snapshot of the CreateCharacter Component", () => {
    const CreateCharacterComponentSnapshot = renderer
        .create(<CreateCharacter />)
        .toJSON();
    expect(CreateCharacterComponentSnapshot).toMatchSnapshot();
});
