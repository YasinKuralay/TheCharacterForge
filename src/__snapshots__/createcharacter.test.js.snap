// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the previous snapshot of the CreateCharacter Component 1`] = `
<div>
  0
</div>
`;
